src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PurposesApiServiceImpl.scala

```diff
@@ -340,4 +340,30 @@ final case class PurposesApiServiceImpl(
       }
     }
   }
+
+  override def createPurpose(
+    q: Option[String],
+    eservicesIds: String,
+    consumersIds: String,
+    producersIds: String,
+    states: String,
+    offset: Int,
+    limit: Int,
+    purposeSeed: PurposeSeed
+  )(implicit
+    contexts: Seq[(String, String)],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
+    toEntityMarshallerCreatedResource: ToEntityMarshaller[CreatedResource]
+  ): Route = {
+    logger.info(s"Creating purpose with seed $purposeSeed")
```

Comment:
as a rule of thumb, do not blindly log the whole seed as it may contain sensitive data.
For example, the risk analysis in the seed can be one of those cases